Cargo.toml

```diff
@@ -50,9 +51,14 @@ url = { version = "2", optional = true }
 
 
 # server:
+time = { version = "0.3", optional = true }
```

Comment:
I'm not sure why this is being added under `server`? Are you actually using this at all?

Reply:
I added the commit `Work around build failure in time v0.3.34 (<-- cookie <-- actix-web)` to work around this build failure that appeared with Rust 1.80:

```
$ ./check.sh
+ cargo check --workspace --no-default-features --all-targets
    Checking legion_prof_viewer v0.3.0 (/home/evanr/sbf/legion-prof-viewer)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.50s
+ cargo check --workspace --no-default-features --features client --all-targets
    Checking legion_prof_viewer v0.3.0 (/home/evanr/sbf/legion-prof-viewer)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.56s
+ cargo check --workspace --no-default-features --features server --all-targets
   Compiling time-macros v0.2.17
    Checking time v0.3.34
error[E0282]: type annotations needed for `Box<_>`
  --> /home/evanr/.cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.34/src/format_description/parse/mod.rs:83:9
   |
83 |     let items = format_items
   |         ^^^^^
...
86 |     Ok(items.into())
   |              ---- type must be known at this point
   |
help: consider giving `items` an explicit type, where the placeholders `_` are specified
   |
83 |     let items: Box<_> = format_items
   |              ++++++++

For more information about this error, try `rustc --explain E0282`.
error: could not compile `time` (lib) due to 1 previous error
```

Rust 1.81 says this:

```
error[E0282]: type annotations needed for `Box<_>`
  --> /home/evanr/.cargo/registry/src/index.crates.io-6f17d22bba15001f/time-0.3.34/src/format_description/parse/mod.rs:83:9
   |
83 |     let items = format_items
   |         ^^^^^
...
86 |     Ok(items.into())
   |              ---- type must be known at this point
   |
   = note: this is an inference error on crate `time` caused by an API change in Rust 1.80.0; update `time` to version `>=0.3.35` by calling `cargo update`

For more information about this error, try `rustc --explain E0282`.
```

So I ran `cargo update` which updated the `Cargo.lock` file. It also resolved this error. However, it exposed a new one.

```
$ ./check.sh
+ cargo check --workspace --no-default-features --all-targets
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.30s
+ cargo check --workspace --no-default-features --features client --all-targets
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.18s
+ cargo check --workspace --no-default-features --features server --all-targets
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.13s
+ cargo check --workspace --all-features --all-targets
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.14s
+ cargo check --workspace --no-default-features --lib --target wasm32-unknown-unknown
    Checking eframe v0.25.0
error[E0308]: mismatched types
   --> /home/evanr/.cargo/registry/src/index.crates.io-6f17d22bba15001f/eframe-0.25.0/src/web/mod.rs:199:16
    |
199 |         if let Some(clipboard) = window.navigator().clipboard() {
    |                ^^^^^^^^^^^^^^^   ------------------------------ this expression has type `Clipboard`
    |                |
    |                expected `Clipboard`, found `Option<_>`
    |
    = note: expected struct `Clipboard`
                 found enum `std::option::Option<_>`

For more information about this error, try `rustc --explain E0308`.
error: could not compile `eframe` (lib) due to 1 previous error
```

I would definitely like a better way to avoid this problem.

Reply:
I think you can remove this line and do:

```
cargo update time
```

And it will update the contents of the `Cargo.lock` file to use the latest version.

Reply:
That seemed to work.